04-Anatomy_of_ASDF_Internals/04-Anatomy_of_ASDF_Internals.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p>Mention memory mapping and that this can be turned off?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>